action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Run tflint with reviewdog'
+description: '🐶 Run tflint with reviewdog on pull requests to enforce best practices'
+author: 'vlaaaaaaad (reviewdog)'
+
+inputs:
+  github_token:
+    description: 'GITHUB_TOKEN'
+    required: true
+  args:
+    description: 'List of arguments to send to tflint'
+    default: '--module --format=checkstyle'
```

Comment:
If users change the args and forgot to include `--format`, this action will be broken and I'd suggest that specify `--format=checktyle` in entrypoint.sh and add a warnign in the description to warn users that they should not change `--format` flag.

Reply:
Some people might want to use another format too so I don't think I'd help by enforcing it.

For example, I tried to use `--format=json` to get the link which is ignored with `--format=checkstyle` and failed fabulously, but someone else might figure it out.

A warning is definitely helpful so I am adding that.

Reply:
Nope, that's false. I am enforcing `reviewdog -f=checkstyle` and making that an option seems like too much.

I think leaving it like this is the best option for now.

Reply:
If someone else figured it out in the future, we need to bump up major version and also we need to warn users to change the flag to remove or modify --format flag if we leave it as is.

If we put `--format` flag in entrypoint.sh, we can just change the flag along with reviewdog `-f` flag and it won't be breaking changes and we can just bump minor/patch version.

Reply:
Agreed!

Added, tested and confirmed working with the example in `README`.